crates/aptos-metrics-core/src/avg_counter.rs

```diff
@@ -0,0 +1,65 @@
+
+// Copyright © Aptos Foundation
+
+use prometheus::{
+    register_counter, register_int_counter, Counter, IntCounter,
+};
+
+
+pub struct AverageCounter {
```

Comment:
What's the issue with using Histogram? Is this to avoid the extra overhead associated with it? Not sure if that's too costly.

Reply:
Yes, makes sense - in fact, we do tune the buckets in many places. But if you care about just sum and average, the histogram should be accurate right?